pretalx-analysis/Which tutorials have been reviewed so far.ipynb

```diff
@@ -33,7 +33,7 @@
     "import numpy as np\n",
     "import pandas as pd\n",
     "import requests as rq\n",
-    "from tqdm.auto import trange"
+    "from tqdm import from tqdm"
```

Comment:
Either I'm about to learn some new Python syntax, or the second from is wrong. Now, I am indeed using the `tqdm` function instead of `trange`, so that's a right fix. Changing `tqdm.auto` back to just `tqdm` has the unfortunate effect of forcing the output back to text-mode TQDM. By using `tqdm.auto`, the distinguished notebook user gains the GUI mode of TQDM, which pleases the eye.

```suggestion
    "from tqdm.auto import tqdm"
```

Reply:
doh... yes, the second from is wrong.